integration-tests/support/src/main/java/org/apache/shiro/testing/web/AbstractContainerIT.java

```diff
@@ -109,6 +128,26 @@ protected WebAppContext createdWebAppContext() throws Exception {
             }
         };
 
+        Server server = jetty.getDelegate();
+
+        // TLS
+        tlsPort = getFreePort();
+
+        final SslContextFactory sslContextFactory = new SslContextFactory.Server();
+        sslContextFactory.setKeyStorePath(TEST_KEYSTORE_PATH.getAbsolutePath());
+        sslContextFactory.setKeyStorePassword(TEST_KEYSTORE_PASSWORD);
+        sslContextFactory.setKeyManagerPassword(TEST_KEYSTORE_PASSWORD);
+
+        HttpConfiguration https = new HttpConfiguration();
+        https.addCustomizer(new SecureRequestCustomizer());
+
+        final ServerConnector httpsConnector = new ServerConnector(
+                server,
+                new SslConnectionFactory(sslContextFactory, HttpVersion.HTTP_1_1.asString()),
```

Comment:
Any reason why we would explicitly don't want http/2 (e.g. for java 9+ builds)?

Reply:
+💯 
I gave it a quick try, but other jetty modules need to be included to support http/2. That can be addressed in the future